db/migrate/20200508155813_populate_response_cached_json.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class PopulateResponseCachedJson < ActiveRecord::Migration[5.2]
+  # Disable transaction wrapper so that the update queries will be committed even if we
+  # have to fail the transaction part-way through.
+  disable_ddl_transaction!
+
+  def up
+    responses = Response.where(filters)
+    remaining_responses = ENV["FORCE_REDO"] ? responses : responses.where(cached_json: nil)
+    cache_responses(remaining_responses)
+  end
+
+  def filters
+    if ENV["SUBSET"]
+      {
+        mission_id: %w[].map do |compact_name|
```

Comment:
Not sure what is going on here. `[].map` anything is just `[]` right?

Reply:
Yes this is just a placeholder (was useful before for demo and may be useful during the deploy migrations but currently is blank behind an env flag)